demo/lib/vm.bash

```diff
@@ -30,6 +30,12 @@ vms:
         grep -E -v '^ *$'
 }
 
+vm-bootstrap() {
+    distro-bootstrap-commands | vm-pipe-to-file "./e2e-bootstrap.sh"
```

Comment:
I'm not able to emulate all the shell levels involved in my head to follow/tell, so it's better if I ask: does `distro-bootstrap-commands` put in enough protection for the recently added code with shell variable evaluation, so that the variables get evaluated only in `e2e-bootstrap.sh` on the host and not before ?

Reply:
Yes, the variables are evaluated only inside the e2e-bootstrap.sh script. But that just relies to the fact that the user escapes the relevant shell variables properly.

Reply:
Okay, so I guess that means that in the current (fedora) `distro-bootstrap-commands()` the 'user' had escaped the variables correctly... ;-)